packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SentenceScopeHandler/SentenceSegmenter.ts

```diff
@@ -4,7 +4,7 @@ import { MatchedText, matchRegex, testRegex } from "../../../../util/regex";
 // A sentence starts with a letter
 const leadingOffsetRegex = /\p{L}/u;
 // A line with no letters is invalid and breaks sentences
-const invalidLineRegex = /(\n[^\p{L}]*\n)/gu;
+const invalidLineRegex = /(\n[^\p{L}]*\n)|(?<=[.!?])(\s*\r?\n)/gu;
```

Comment:
this regex confuses me a bit:

- the first part is matching an entire line including newlines but the second part is not
- the second part is handling `\r` but the first part never mentions it

I'm not saying it's necessarily incorrect, but I don't understand it.  Would it be possible to fix and/or update the comment?

Reply:
better?